### PR TITLE
docs: revisit configuration auditing

### PR DESCRIPTION
**Pull request by danielpacak**
This PR is getting a bit wild and out of original scope, but I hope you enjoy it ;)

Resolves: #1019

Signed-off-by: Daniel Pacak <pacak.daniel@gmail.com>

### CODE REVIEW

File: docs/configuration-auditing/pluggable-scanners/index.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
-# Configuration Checkers
+# Pluggable Scanners
+
+!!! warning
+    Pluggable configuration audit scanners are being deprecated. They did great job when we first integrated them with
```
Comment (chen-keinan) [inline docs/configuration-auditing/pluggable-scanners/index.md:4]:
```suggestion
    Pluggable configuration audit scanners are being deprecated. They did a great job when we first integrated them with
```

File: docs/configuration-auditing/pluggable-scanners/index.md [hunk 2]
```diff
@@ -1,4 +1,9 @@
...
+    Starboard, but recently we found that they are not very efficiency and customizable. Therefore, we are switching
```
Comment (chen-keinan) [inline docs/configuration-auditing/pluggable-scanners/index.md:5]:
```suggestion
    Starboard, but recently we found that they are not very efficient and customizable. Therefore, we are switching
```

File: docs/configuration-auditing/index.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+improperly creates a high-impact security and compliance risk. DevOps, and platform owners need the ability to
+continuously assess build artifacts, workloads, and infrastructure against configuration hardening standards to
+remediate any violations.
+
+Starboard configuration audit capabilities are purpose-built for Kubernetes environments. In particular, Starboard
+Operator continuously monitors images, workloads, and Kubernetes infrastructure components against common configuration
+security standards and generates detailed assessment reports, which are then stored in the default Kubernetes database.
+
+Kubernetes applications and other core configuration objects, such as Ingress, NetworkPolicy, ResourceQuota, RBAC
+resources, are evaluated against [Built-in Policies]. Beyond that, cluster nodes are constantly assessed against CIS
```
Comment (chen-keinan) [inline docs/configuration-auditing/index.md:14]:
```suggestion
resources, are evaluated against [Built-in Policies]. Beyond that, cluster nodes are constantly assessed against the CIS
```

**Review by chen-keinan — CHANGES_REQUESTED** [review]

File: docs/configuration-auditing/infrastructure-scanners/index.md [hunk 4]
```diff
@@ -11,12 +11,15 @@ The CIS benchmark for Kubernetes provides prescriptive guidance for system and a
...
 
 ```
 starboard scan ciskubebenchreports
 ```
 
+In both cases, results are stored as [CISKubeBenchReport] resources controller by the corresponding cluster node.
+
 If everything goes fine, list benchmark results with the `kubectl get` command:
 
 ```console
```
[the remark this answers is not in the document]
Comment (danielpacak) [reply docs/configuration-auditing/infrastructure-scanners/index.md:25]: Console has special purpose here: It interpreter the dollar ($) prefix and highlights the first line

File: docs/configuration-auditing/built-in-policies.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
+# Built-in Configuration Audit Policies
```
Comment (danielpacak) [inline docs/configuration-auditing/built-in-policies.md:1]: @liamg we need a script to generated such markdown from defsec repo 😄

**Review by chen-keinan — APPROVED** [review]

**Pull request merged**